public abstract class Osoba {
    private String jmeno;

    public Osoba(String jmeno) {
        this.jmeno = jmeno;
    }

    public String getJmeno() {
        return jmeno;
    }

    public void setJmeno(String jmeno) {
        this.jmeno = jmeno;
    }

    public abstract void popis();
}
